src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -13162,481 +13162,13 @@ def test_aks_update_upgrade_settings(self, resource_group, resource_group_locati
             checks=[self.is_empty()],
         )
 
+    # ACNS related tests
     @AllowLargeResponse()
     @AKSCustomResourceGroupPreparer(
         random_name_length=17,
         name_prefix="clitest",
         location="westcentralus",
     )
-    def test_aks_update_advanced_network_observability(
```

Comment:
Please also remove the corresponding recording files (`<test case name>.yaml`) from `src/aks-preview/azext_aks_preview/tests/latest/recordings`.

Reply:
Removed